### DISCUSSION TITLE
Release cadence

### DISCUSSION THREAD

**Discussion by jittery-droid** [jackc/pgx#1767] [Q&A] [answered]
Is there a regular release cadence? I am primarily wanting access to [this](https://github.com/jackc/pgx/pull/1718) feature.

**Comment by jackc** [comment, marked as answer]
Urgent bug fixes typically get tiny releases immediately or within a week or so.

As a rule of thumb, I do a minor release when there are several new features. I also try to wait at least 3 months between minor releases. I'm trying to minimize the time pgx users spend testing and validating pgx upgrades in their applications.

There are no plans for another major release.

---

I expect v5.5.0 to be released within a few weeks.

**Reply by jittery-droid** [reply]
Thank you for the reply @jackc that makes perfect sense.

**Comment by tkgalk** [comment]
Any updates on 5.5.0? https://github.com/jackc/pgx/pull/1718 is really blocking us, and would be amazing to have released and be able to use without faffing around with specific commits. :)

**Reply by jackc** [reply]
v5.5.0 was just released.
